<launch>
	<node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen" args="-d $(find ct_ros_nodes)/rviz/hyqViz.rviz"/>
	<param name="hyq_description" command="$(find xacro)/xacro.py '$(find ct_models)/urdf/HyQ/hyqb.urdf.xacro'"/>
	
	<node name="hyq_simulation" pkg="ct_ros_nodes" type="HyQSimulator" respawn="false" output="screen" >
	  <param name="useProjectedDynamics" value="true" />
	</node>
	
	<node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
	
	<node pkg="robot_state_publisher" type="robot_state_publisher" name="hyq_state_publisher"  >
      <remap from="robot_description" to="hyq_description" />
    </node>
	
</launch>